Description: Enable tracker full text search
 Tracker in Debian is compiled with FTS support, so we make use of that in
 GTK3's file selector dialog.
 This way files can be found by their content and not only their file name.

Index: gtk+3.0-3.3.20/gtk/gtksearchenginetracker.c
===================================================================
--- gtk+3.0-3.3.20.orig/gtk/gtksearchenginetracker.c	2012-03-18 21:44:22.000000000 +0100
+++ gtk+3.0-3.3.20/gtk/gtksearchenginetracker.c	2012-03-20 18:48:18.113061457 +0100
@@ -50,7 +50,7 @@
  * use filename matching instead. This doesn't use the content of the
  * file however.
  */
-#undef FTS_MATCHING
+#define FTS_MATCHING
 
 /*
  * GtkSearchEngineTracker object
